sdk/tables/aztables/recordings/TestServiceClient_Cosmos/TestQueryTable-variables.yaml

```diff
@@ -0,0 +1,5 @@
+---
+COSMOS_TABLES_ENDPOINT_SUFFIX: cosmos.azure.com
+TABLES_COSMOS_ACCOUNT_NAME: chrisstablesprim
+TABLES_PRIMARY_COSMOS_ACCOUNT_KEY: Kg==
+randomSeed: "1617573294"
```

Comment:
So these are also sanitized when calling the default sanitizer?

Reply:
These are sanitized only if `GetRecordedVariable` is called with a `VariableType` argument that indicates it is a secret.

Reply:
Ah got it, thanks for the info :)